### PR TITLE
Reading Local Files

### LINKED ISSUE #1: Read local file
Read the local file and print its contents

### PR DESCRIPTION
**Pull request by brandonbush**
Resolves #1

### CODE REVIEW

**Comment by brandonbush** [comment]
@Jon-Call please review

File: src/testruns.py [hunk 1]
```diff
@@ -45,6 +45,6 @@ def main(**kwargs):
 print(tswizzle(31))
 
 #Get a file from this directory
-# with open('testfile.txt', 'r') as f:
```
Comment (Jon-Call) [inline src/testruns.py:48]: Could you use more descriptive instead of single letter variable names? This will help make the code more readable.

**Review by Jon-Call — CHANGES_REQUESTED** [review]

File: src/testruns.py (hunk 1 above)
Comment (Jon-Call) [reply src/testruns.py:48]: also select the line below the code you want to comment on when reviewing, that way it shows up in the PR discussion

Comment (brandonbush) [reply src/testruns.py:48]: Fixed in https://github.com/brandonbush/test-git-commands/pull/5/commits/774de643729f011abbfab48781d2188dab280252

**Review by Jon-Call — APPROVED** [review]

**Pull request merged**